phoenix-tracing-webapp/src/build/trace-server-runnable.xml

```diff
@@ -43,7 +43,19 @@
       <outputDirectory>/</outputDirectory>
       <includes>
         <include>org.apache.phoenix:phoenix-tracing-webapp</include>
```

Comment:
Ideally, this should be done by the shade plugin instead. (Not required for this patch, just a remark)